app/services/cache/MongoCacheClient.scala

```diff
@@ -184,6 +189,35 @@ class MongoCacheClient(appConfig: AppConfig, db: () => DefaultDB, applicationCry
       case _ => None
     }
 
+  def findWithCacheMiss[T](id: String, newId: String, key: String)(implicit reads: Reads[T]): Future[Option[T]] = {
```

Comment:
Why named this?

Reply:
relic from first implementation, will update